test/cmd/newapp.sh

```diff
@@ -18,6 +18,15 @@ os::cmd::expect_failure 'oc new-app unknownhubimage -o yaml'
 os::cmd::expect_failure_and_text 'oc new-app docker.io/node~https://github.com/openshift/nodejs-ex' 'the image match \"docker.io/node\" for source repository \"https://github.com/openshift/nodejs-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_failure_and_text 'oc new-app https://github.com/openshift/rails-ex' 'the image match \"ruby\" for source repository \"https://github.com/openshift/rails-ex\" does not appear to be a source-to-image builder.'
 os::cmd::expect_success 'oc new-app https://github.com/openshift/rails-ex --strategy=source --dry-run'
+# When env file doesn't exist
+# This is for checking if file exists and then delete
+# Should we create a UUID for the file we are generating?
+if [ -f '/tmp/sampleabc' ] ; then rm /tmp/sampleabc; fi
```

Comment:
use mktemp to create an empty directory and reference a non-existent file inside that dir, don't erase potentially existing files off people's machines.